### PR TITLE
cmd: fix raft engine ctl

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### What is changed and how it works?

Issue Number: Ref #11119

What's Changed:

```commit-message
None
```

### Check List 

- No code

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by tabokie** [comment]
Ping @Connor1996 @5kbpers

File: cmd/tikv-ctl/src/cmd.rs [hunk 1]
```diff
@@ -548,7 +548,11 @@ pub enum Cmd {
         version: u64,
     },
     /// Control for Raft Engine
-    RaftEngineCtl { args: Vec<String> },
+    /// Usage: tikv-ctl raft-engine-ctl -- --help
+    RaftEngineCtl {
```
Comment (Connor1996) [inline cmd/tikv-ctl/src/cmd.rs:552]: do you update the doc?

Comment (tabokie) [reply cmd/tikv-ctl/src/cmd.rs:552]: You mean pingcap/docs? I haven't. raft-engine-ctl is a bit complicated in itself, I may publish it to docs after I (or someone else) creates a dedicated troubleshooting.md for it.

**Review by Connor1996 — APPROVED** [review]

**Review by glorv — APPROVED** [review]

**Pull request merged**